extensions/prefetch/k8s.go

```diff
@@ -41,8 +43,11 @@ spec:
         app: prefetch-{{.Number}}
     spec:
       initContainers:
+        # return container is used to pass the return application to other containers.
+        # containers use this to exit immediately after loading the image.
+        # See: https://github.com/glazychev-art/return
         - name: return
-          image: rrandom312/return
+          image: ghcr.io/glazychev-art/return
```

Comment:
Since we're using this in the project, should we move it to the nsm org?

Reply:
Done